vue-app/package.json

```diff
@@ -15,6 +15,7 @@
     "@kleros/gtcr-encoder": "^1.4.0",
     "@openfonts/inter_all": "^1.0.2",
     "@walletconnect/web3-provider": "^1.5.1",
+    "approximate-number": "^2.1.0",
```

Comment:
Curious you're thinking behind choosing this package - seems like it's not widely used & hasn't been active much since 2015-2017: https://github.com/nfriedly/approximate-number/graphs/contributors

Reply:
e.g. why not just use the `Intl.NumberFormat` - can it not achieve the same purpose? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat

That's what we use for ethereum.org, e.g. https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/StatsBoxGrid.js#L181
We use the polyfill on ethereum.org https://www.npmjs.com/package/@formatjs/intl-numberformat
Looks like this has 100Ks weekly downloads vs 1Ks.

Reply:
@samajammin Yes, this package does have what we need, though an issue I encountered with it was with our version of TypeScript not supporting the `notation: 'compact'` option on the `NumberFormatOptions` type ([see this issue on the TypeScript repo](https://github.com/microsoft/TypeScript/issues/40038)). 
This is what enables the abbreviated numbers (ie. 1234 => "1.23K"). It appears to have been [fixed](https://github.com/microsoft/TypeScript/pull/40039) since then, but upgrading TypeScript is likely going to come with other breaking changes. 

I do agree it would be nicer to use this library, could look into upgrading TypeScript...just don't want to open a can of worms.